src/languages/html.coffee

```diff
@@ -73,13 +73,23 @@ module.exports = {
             'object', 'output', 'progress', 'q', 'ruby', 's', 'samp', 'select', 'small',
             'span', 'strong', 'sub', 'sup', 'svg', 'template', 'textarea', 'time', 'u', 'var',
             'video', 'wbr', 'text',
-            'acronym', 'address', 'big', 'dt', 'ins', 'small', 'strike', 'tt',
-            'pre',
-            'h1', 'h2', 'h3', 'h4', 'h5', 'h6'
+            'acronym', 'address', 'big', 'dt', 'ins', 'strike', 'tt'
         ]
       items:
         type: 'string'
-      description: "List of tags (defaults to inline) that should not be reformatted"
+      description: "List of inline tags"
+    content_unformatted:
+      type: 'array'
+      default: [ 'pre', 'textarea' ]
+      items:
+        type: 'string'
+      description: "List of tags whose contents should not be reformatted"
+    unformatted:
+      type: 'array'
+      default: []
```

Comment:
No,  `inline` is an entire new behavior.  `Unformatted` should generally behave the same, but inline tags more accurately model correct HTML behavior.  You have sample tests... But I can make sense of which ones should be failing or not...

Reply:
> Could we leave the existing `unformatted` without changes then?

No, please don't do that. That would be partially defeating the purpose of `inline`, because you'll be saying not to format things like `<a>` at all!! And no one wants that. We only had it not being formatted before because it would format it as "not inline". This has been fixed.

> … a breaking change of unformatted for those users who do actually use the default value …

But _nobody_ wants the old `unformatted` default values! Nobody says, "I like my inline elements the way they are; keep your hands off." And if there is that one person, they can add `"unformatted"` to their settings.

But otherwise, to get inline elements to be formatted, if you leave the existing `unformatted` settings everyone (the 99.997% of people who want them to be formatted) will have to add `"unformatted": []` to their manual settings!!

Reply:
@Glavin001  I agree with Garret that keeping unformatted will make inline largely ineffective. But in the interest of getting the new options added and available I'm happy to keep the existing default in this version, if that is what it takes. I understand the need to limit the risk of breaking changes.

Reply:
> But otherwise, to get inline elements to be formatted, if you leave the existing unformatted settings everyone (the 99.997% of people who want them to be formatted) will have to add "unformatted": [] to their manual settings!!

@bitwiseman Do you also expect everyone to be using `"unformatted": []` from now on? Is there *no value* in `unformatted` anymore and going to be removed from `js-beautify` later? Given the new ideal value for `unformatted` is `[]` it sounds like we should remove the `unformatted` option entirely in favour of `inline` and `content_unformatted` which "more accurately model correct HTML behavior".

Fortunately, `unformatted` is only used by `JS Beautify` beautifier: https://github.com/Glavin001/atom-beautify/blob/master/docs/options.md#unformatted
So if we really need to make a breaking change, let's do it.
If the expectation is `unformatted` should no longer be used, let's not only go half way and set it to `[]`, let's remove it entirely and make a big note in the changelog about these new and improved options for formatting.